L10nSnapshotTests/L10nPermissionStringsSnapshotTests.swift

```diff
@@ -0,0 +1,9 @@
+//
+//  L10nPermissionStringsSnapshotTests.swift
+//  Client
+//
+//  Created by Stefan Arentz on 2016-11-07.
+//  Copyright © 2016 Mozilla. All rights reserved.
+//
+
```

Comment:
Oh odd. These two files should not be in this PR actually. That is what the followup PR is for. I'll try to remove them.